tests/unit/modules/network/ios/test_ios_logging_global.py

```diff
@@ -718,3 +719,20 @@ def test_ios_logging_global_replaced_idempotent(self):
 
         self.maxDiff = None
         self.assertEqual(sorted(result["commands"]), sorted(replaced))
+
+    def mock_connection(self):
+        """Fixture to provide a mock connection object"""
+        mock = MagicMock()
+        return mock
+
+    def test_get_logging_data_logging_trap_informational(self):
```

Comment:
so what we need to test here is tap: information being part of gathered and merged operation.